test/factories/years.rb

```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
   factory :year do
-    sequence(:year) { |n| 2000 + n }
+    sequence(:name) { |n| (2000 + n).to_s }
```

Comment:
I think we want our factory to look like a real school year as well so we could use some interpolation here and have `(2000 + n) -  (2000 + n+1)` or something like that so the string will represent what we should expect them to look like.